proto/cli.proto

```diff
@@ -1160,4 +1166,31 @@ message CreateVolumeResponse {
 	}
 	Result result = 3;
 	PosInfo info = 4;
+}
+
+message SetVolumePropertyRequest {
+	string command = 1;
+	string rid = 2;
+	string requestor = 3;
+	message Param {
+		string name = 1;
+		string array = 2;
+		string newVolumeName = 3;
+		uint64 size = 4;
+		uint64 maxiops = 5;
+		uint64 maxbw = 6;
+		bool updateprimaryvol = 7;
```

Comment:
What is the use of this parameter? What is the difference between isprimarywalvol and updateprimarywalvol

Reply:
Because of the limitation of protobuf, it is impossible to know if the field is valid (i.e., this value is input by the user). updateprimaryvol field indicates if the server needs to update the isprimarywalvol.